geoai/osd.py

```
@@ -0,0 +1,137 @@
+                        if identifier == "LeakyReLU":
+                            return orig("leaky_relu")
+                        return orig(identifier)
+
+                    _original_getters.append((m.activations, "get", orig_get))
+                    patched_targets.add((id(m.activations), "get"))
+                    m.activations.get = patched_get
+        _patch_ref_count += 1
+
+    original_imread = tifffile.imread
+
+    def patched_imread(*args, **kwargs):
```

Ruff (0.15.21)

[warning] Missing return type annotation for private function `patched_imread`

(ANN202)